torch/csrc/dynamo/cpython_defs.c

```diff
@@ -0,0 +1,137 @@
+#include "cpython_defs.h"
```

Comment:
Naming nit - `torch/csrc/dynamo/cpython_defs.c` is a weird file name, but what we are doing here is also weird.